site/_docs/configuration.md

```diff
@@ -180,7 +180,7 @@ class="flag">flags</code> (specified on the command-line) that control them.
         <p class="description">Enable auto-regeneration of the site when files are modified.</p>
       </td>
       <td class="align-center">
-        <p><code class="flag">-w, --watch</code></p>
+        <p><code class="flag">-w, --watch, --no-watch</code></p>
```

Comment:
Sure, will update.

Out of curiosity, is there a short equivalent of the `--no-watch` flag? Seems odd to only have `-w`.